landingPages/initialLaunch/index.html

```diff
@@ -1,14 +1,93 @@
 <!DOCTYPE html>
 <html>
- <head>
-  <meta charset="UTF-8">
-  <title>title</title>
-  <meta name="author" content="name">
-  <meta name="description" content="description here">
-  <meta name="keywords" content="keywords,here">
-  <link rel="shortcut icon" href="favicon.ico" type="image/vnd.microsoft.icon">
- </head>
- <body>
-
- </body>
-</html>
\ No newline at end of file
+<head>
+	<meta charset="UTF-8">
+	<title>Kevin Smith Transportation Group</title>
+	<meta name="description" content="Philadelphia region's newest luxury limoursine and total transportation service.">
+	<meta name="keywords" content="luxury, limo, transportation, Philadelphia">
+	<link rel="shortcut icon" href="favicon.ico" type="image/vnd.microsoft.icon">
+	<link href='http://fonts.googleapis.com/css?family=Open+Sans:400italic,400,700' rel='stylesheet' type='text/css'>
+	<link rel="stylesheet" type="text/css" href="css/reset.css">
+	<link rel="stylesheet" type="text/css" href="css/stylesheet.css">
+</head>
+<body id="landing-page">
+	<header>
+		<div class="header-logo">
+			<img src="images/ktsg-logo.png" alt="Kevin Smith Transportation Group" height="150" width="288">
+		</div>
+	</header>
+
+	<section class="grid">
+		<div class="headline">
+			<h1>Your Luxury Limousine and Total Transportation Service.</h1>
+			<h2 class="h2-center">Arriving July 2015.</h2>
+			<p class="headline-copy">The Kevin Smith Transportation Group is the Philadelphia region's newest luxury limousine and total transportation service. We hope to change the way you think about limousine, airport pickup, and towncar service by providing you with a customized experience you'll love.</p>
+		</div>
+		<div class="banner-container">
+			<div class="banner row">
+				<figure class="col-1-4">
+					<img src="images/corporate.png" alt="Corporate" height="230" width="280">
```

Comment:
Not crazy about the way these look when resizing the browser, even slightly. I don't think you need to ensure it looks good at every possible pixel size, but if someone is looking at it on even a slightly smaller screen size, the margins disappear.

One thing to consider is giving the images a class and adding `max-width: 100%` to them. That will keep their widths to 100% of the parent container thus keeping the margin. Try it out and resize the browser--think you might like the way it looks better. :smiley: